server/jetstream_cluster_4_test.go

```diff
@@ -1218,13 +1225,21 @@ func TestJetStreamClusterStreamOrphanMsgsAndReplicasDrifting(t *testing.T) {
 				msets = append(msets, mset)
 			}
 			for seq := state.FirstSeq; seq <= state.LastSeq; seq++ {
+				var expectedErr error
 				var msgId string
 				var smv StoreMsg
 				for _, mset := range msets {
 					mset.mu.RLock()
 					sm, err := mset.store.LoadMsg(seq, &smv)
 					mset.mu.RUnlock()
-					require_NoError(t, err)
+					if err != nil || expectedErr != nil {
```

Comment:
What error is ok here? Should we add a comment? Was a bit tough following this logic.

Reply:
Added comments to explain what's going on, but basically it's trying to prove that each replica reports the same error (or non-error) for any given sequence.